tools/ot/src/test/java/dk/alexandra/fresco/tools/ot/base/TestFunctionalNaorPinkas.java

```diff
@@ -48,23 +48,18 @@ public void shutdown() {
     testRuntime.shutdown();
   }
 
-  private List<Pair<StrictBitVector, StrictBitVector>> otSend(int iterations)
-      throws Exception {
-    Network network = new CheatingNetwork(
-        RuntimeForTests.defaultNetworkConfiguration(1, Arrays.asList(1, 2)));
+  private List<Pair<StrictBitVector, StrictBitVector>> otSend(int iterations) throws Exception {
+    Network network =
+        new CheatingNetwork(RuntimeForTests.defaultNetworkConfiguration(1, Arrays.asList(1, 2)));
     try {
       Drbg rand = new AesCtrDrbg(HelperForTests.seedOne);
-      DhParameters params = new DhParameters();
-      Ot otSender = new NaorPinkasOt(2, rand, network, params
-          .computeSecureDhParams(1, 2, rand, network));
```

Comment:
Do we want to remove the option of computing computeSecureDhParams from DhParameters? Because if not, then the testing using this method should not have been removed from all the tests in this class. On the other hand if we do want to remove the computeSecureDhParams  option then the code for this method should be removed from DhParameters.

Reply:
Good point, I will go with just removing the computeSecureDhParams method, as we are never really using it anyway.